giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/files/IFilesManager.java

```diff
@@ -68,5 +68,7 @@
     public abstract Map<String, Map<String, String>> getUploadedFilenames(String username);
 
     public abstract IFile getFileByRequestId(String requestId);
+    
+    public abstract boolean changeDocumentAccess(IDocument doc, DocumentAccess docAccess) throws UnstorableObjectException;
```

Comment:
this should return "true" when action was executed successfully, otherwise false. The way it is right now is not very intuitive.